src/app.coffee

```diff
@@ -62,6 +63,9 @@ class App
                     deviceName: deviceName
                     password: password
                 Devices.registerDevice options, next
+                filterManager = new FilterManager cozyUrl, deviceName, password
+                filterManager.createOrUpdate {file: true, folder: true}, ->
```

Comment:
It's the first parameter. If you want to use 2 objects for 2 parameters (which is often not a good idea), you have to use `{` and `}`:

```coffee
filterManager.createOrUpdate {file: true}, {folder: true}, ->
```